Guide_1_Send_Payment.md

```diff
@@ -1,31 +1,52 @@
 # Guide: Making a Payment
 
-_(Note: Are there any specific steps required for different payment schemes (SWIFT etc.)? Do the steps in this guide work the same for any scheme? Should be mentioned somehow)_
+_**(Note: Are there any specific steps required for different payment schemes (SWIFT etc.)? Do the steps in this guide work the same for any scheme? Should be mentioned somehow)**_
```

Comment:
Suggest that we just use the FPS (faster payments) example. Once we support other payment types, we can add a separate tutorial that explains how to send different payment types, and how they are routed etc.